lib/path.js

```diff
@@ -540,6 +544,42 @@ const win32 = {
     if (from === to)
       return '';
 
+    if (fromOrig.length !== from.length || toOrig.length !== to.length) {
+      const fromSplit = StringPrototypeSplit(fromOrig, '\\');
+      const toSplit = StringPrototypeSplit(toOrig, '\\');
+      if (fromSplit[fromSplit.length - 1] === '') {
+        fromSplit.pop();
+      }
+      if (toSplit[toSplit.length - 1] === '') {
+        toSplit.pop();
+      }
```

Comment:
I don't think we need the `.pop()` calls, we could simply decrement `fromLen` and `toLen` by one, no?

Reply:
Thank you for your review and suggestion. I implemented the change as you recommended, but unfortunately, it didn't impact the benchmark results. I think the reason is that the time complexity of `Array.pop(1)` is O(1).

Reply:
Performance gains would be possible by not splitting the input and instead looking through the strings for each separators and lowercaseing those parts. That way the array creation is not needed and the result could be put together on the fly. That's just more complex code and I would concentrate on landing this as is before optimising it for performance.